Update/doc (#571)
algolia/docsearch: docs/src/recommended-configuration.md

---
layout: two-columns

> title: Recommendation

Reviewer: What about `Best Practices`?

selectors outside this element will be `global`. Every selectors appear in the
same order than their `lvl` along the HTML flow.

### Introduces global information as meta tags

Our crawler automatically extract information from our DocSearch specific meta

> taf:

Reviewer: ```suggestion
 tags:
```

```html
<meta name="docsearch:language" content="en">
<meta name="docsearch:version" content="1.0.0">
```

The `content` value of the meta will be added to every records extracted from

> the page. Giving that the name is `docsearch:$NAME`, `$NAME` will be set as a

Reviewer: ```suggestion
 the page. Given that the name is `docsearch:$NAME`, `$NAME` will be set as a
```

> attribute in every of them. Its value will be its related `content`value. You

Reviewer: ```suggestion
 attribute in each of them. Its value will be its related `content` value. You
```

> can then transform them as [`facetFilter`][6] to let you filter over it.

Reviewer: ```suggestion
 can then transform them as [`facetFilters`][6] to filter over them.
```

Look at this example with our vuepress integration:

```json
"custom_settings": {
    "attributesForFaceting": [
      "lang"
    ]
  }
```

> 

Reviewer: You're mentioning `facetFilters` (for the front) above, then here using `attributesForFaceting` (for the settings). I think it needs a bit more clarification.
